defaults/main.yml

```diff
@@ -2,5 +2,10 @@ podman_user_name: osbs-podman
 podman_user_uid: 2022
 podman_user_group_name: osbs-podman
 podman_user_group_gid: 2022
+# These subordinate numbers are picked per doc: https://systemd.io/UIDS-GIDS
+podman_user_subordinate_uid_start: 100000
+podman_user_subordinate_uid_count: 1878948191  # 1879048191 - 100000
```

Comment:
@lkolacek the second number in subuid/subgid line is the count of IDs, so I believe we should use `1878948191`(which is `1878948191-100000`) instead of `1879048191`, because I'm not sure whether ID bigger than `1879048191` (`6FFFFFFF` in hex) can cause any issue. wdyt?

Reply:
There also seems to be a minimum container UID:

UID/GID | Purpose | Defined By | Listed in
-- | -- | -- | --
524288…1879048191 | Container UID ranges | systemd | nss-systemd

Should that be taken into account as well, leaving (1879048191 - 524288 - 100000) for the count?

Reply:
It depends on what the number should start with, count is "max - start", so if we start with 100000, count is 1879048191 -  100000, if we start with 524288, count is 1879048191 - 524288.

Reply:
Oh I see, somehow I thought the starting 100000 would get remapped to 100000 + 524288 automatically which is probably not true :)

Reply:
+1, it's working also with 1879048191 but you're right that it will end up mapped outside of the container UID rang